code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -401,3 +401,17 @@
 
 	R.make_shell(src)
 	return TRUE
+
+/obj/item/borg/upgrade/traitorsec
+	name = "Oppressor upgrade module"
+	desc = "Illegal upgrade board that unlocks the banned security module."

```

Comment:
Also same here, it's a config setting and not explicitly banned.

Should set the descriptions based on config.forbid_secborg being true or false imo